src/pages/ThorChainLP/components/ReusableLpConfirm.tsx

```diff
@@ -0,0 +1,281 @@
+              ? confirmedQuote.runeFiatLiquidityAmount
+              : confirmedQuote.assetFiatLiquidityAmount
+
+          return (
+            <Card
+              display='flex'
+              alignItems='center'
+              justifyContent='center'
+              flexDir='column'
+              gap={4}
+              py={6}
+              px={4}
+              flex={1}
+            >
+              <AssetIcon size='sm' assetId={_asset.assetId} />
+              <Stack textAlign='center' spacing={0}>
+                <Amount.Crypto
+                  fontWeight='bold'
+                  value={amountCryptoPrecision}
+                  maximumFractionDigits={4}
+                  symbol={_asset.symbol}
+                />
+                <Amount.Fiat fontSize='sm' color='text.subtle' value={amountFiatUserCurrency} />
+              </Stack>
+            </Card>
+          )
+        })}
+      </Stack>
+    )
+  }, [
+    asset,
+    baseAsset,
+    confirmedQuote.assetCryptoLiquidityAmount,
+    confirmedQuote.assetFiatLiquidityAmount,
+    confirmedQuote.runeCryptoLiquidityAmount,
+    confirmedQuote.runeFiatLiquidityAmount,
+    divider,
+    pool,
+  ])
+
+  if (!(pool && asset && baseAsset)) return null
+
+  return (
+    <SlideTransition>
+      <CardHeader display='flex' alignItems='center' gap={2}>
+        <Flex flex={1}>
+          <IconButton onClick={handleBack} variant='ghost' aria-label='back' icon={backIcon} />
+        </Flex>
+        <Flex textAlign='center'>{translate('common.confirm')}</Flex>
+        <Flex flex={1}></Flex>
+      </CardHeader>
+      <CardBody pt={0}>
+        <Stack spacing={8}>
+          {depositCards}
+          <Timeline>
+            <TimelineItem>
+              <Row fontSize='sm' fontWeight='medium'>
+                <Row.Label>{translate('pools.chainFee', { chain: 'ShapeShift' })}</Row.Label>
+                <Row.Value>Free</Row.Value>
+              </Row>
+            </TimelineItem>
+            <TimelineItem>
+              <Row fontSize='sm' fontWeight='medium'>
+                <Row.Label>{translate('pools.chainFee', { chain: baseAssetNetwork })}</Row.Label>
+                <Row.Value display='flex' gap={1}>
+                  <Amount.Crypto value='0.02' symbol='RUNE' />
+                  <Flex color='text.subtle'>
+                    {'('}
+                    <Amount.Fiat value='10.00' />
+                    {')'}
+                  </Flex>
+                </Row.Value>
+              </Row>
+            </TimelineItem>
+            <TimelineItem>
+              <Row fontSize='sm' fontWeight='medium'>
+                <Row.Label>{translate('pools.chainFee', { chain: assetNetwork })}</Row.Label>
+                <Row.Value display='flex' gap={1}>
+                  <Amount.Crypto value='0.02' symbol='ETH' />
+                  <Flex color='text.subtle'>
+                    {'('}
+                    <Amount.Fiat value='10.00' />
+                    {')'}
```

Comment:
I think(?) we can do this
```suggestion
                    <Amount.Fiat value='10.00' prefix='(' suffix=')' />
```

Reply:
This adds space, the prefix suffix thing is more for $USD 400, or 400 USD. So we would end up with something like ( $400 ). Instead of ($400).

Reply:
Its not that big of a deal, i'll tackle the extra space in another PR at some point.